src/svelte/Breadcrumbs.svelte

```diff
@@ -83,7 +84,7 @@
   }
 </script>
 
-<nav class="breadcrumbs" aria-label={ariaLabel}>
+<nav class={`breadcrumbs ${classNames}`} aria-label={ariaLabel}>
```

Comment:
Would this be the same?

```suggestion
<nav class="breadcrumbs {classNames}" aria-label={ariaLabel}>
```